<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MathBOX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_wynik"/>
      </item>
      <item row="4" column="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="3">
         <widget class="QPushButton" name="pLn">
          <property name="text">
           <string>ln</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pAsin">
          <property name="text">
           <string>asin</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pAcos">
          <property name="text">
           <string>acos</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="pAbs">
          <property name="text">
           <string>abs</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pMult">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pDiv">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pMinus">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pX">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pPlus">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pEndBracket">
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pPower">
          <property name="text">
           <string>x^n</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pOpenBracket">
          <property name="text">
           <string>(</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pSin">
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pCos">
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pTg">
          <property name="text">
           <string>tg</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pLog">
          <property name="text">
           <string>log</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pTanh">
          <property name="text">
           <string>tanh</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pAsinh">
          <property name="text">
           <string>asinh</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pAcosh">
          <property name="text">
           <string>acosh</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="pExp">
          <property name="text">
           <string>exp</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="4" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="2">
         <widget class="QPushButton" name="pDot">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pEqual">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="p7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="p6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="p8">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="p0">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="p5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="b1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="p9">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="b2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="p4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="p3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pRysuj">
          <property name="text">
           <string>Rysuj</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pPochodna">
          <property name="text">
           <string>Rysuj pochodna</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>&lt;-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="sPowiekszenie"/>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>wiersze:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>46</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>kolumny:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pMatrixA">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Macierz A</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pMatrixB">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>410</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Macierz B</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pMatrixAdd">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pMatrixSub">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pMatrixMul">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sIloscWierszy">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sIloscKolumn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="lMatrixA">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>440</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>niezapisana</string>
      </property>
     </widget>
     <widget class="QLabel" name="lMatrixB">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>440</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>niezapisana</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>410</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pTrans">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>410</y>
        <width>89</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Transpozycja</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
